packages/types/src/interfaces/balances/types.ts

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable */
 
 import type { Struct, i128 } from '@polkadot/types';
-import type { BlockNumber } from '@polkadot/types/interfaces/runtime';
+  import type { BlockNumber } from '@polkadot/types/interfaces/runtime';
 
```

Comment:
This is an auto-generated file. I don't know why `@polkadot/typegen` suddenly does this. This happens in all the generated `types.ts` files.